app/server/routes/api.js

```diff
@@ -250,8 +250,10 @@ module.exports = function(router) {
     // Accept the hacker. Admin only
     var id = req.params.id;
     var user = req.user;
-    UserController.admitUser(id, user, defaultResponse(req, res));
-    UserController.sendConfirmationEmailById(id, function() {});
+    UserController.admitUser(id, user, function(err, user) {
+      defaultResponse(err, user);
```

Comment:
I don't know if defualtResponse allows this but it doesn't seem to spawn an error when everything goes through. So I guess its good